test/verify/check-networking-firewall

```diff
@@ -145,8 +152,8 @@ class TestFirewall(NetworkCase):
         b.wait_present("#add-services-button:enabled")
         b.click("#add-services-button")
         b.wait_present("#cockpit_modal_dialog .list-view-pf input[data-id='pop3']")
-        # rhel-8 comes with an extra preconfigured libvirt zone
-        if m.image not in ["rhel-8-1", "rhel-8-1-distropkg"]:
+        # some images come with an extra preconfigured libvirt zone
+        if m.image not in ["rhel-8-0", "rhel-8-1", "rhel-8-0-distropkg", "rhel-8-1-distropkg", "fedora-31"]:
```

Comment:
This reverts the removal of rhel-8-0*. master stopped supporting that, so these should go. Same below.